### ISSUE TITLE
Identifier Service

### ISSUE THREAD

**Issue by cmh2166** [sul-dlss-labs/taco#175]
- [ ] What id service do we use for identifiers
  - SURI on lyberservices subset of calls
  - ~SURI Rails rewrite / cbeer~
  - ID Proxy stuff (that calls SURI; rewriting might be required)
  - Fake out entirely within TACO codebase
- [ ] Based on the above, get one implemented & connections available to TACO
- [ ] Larger topic (to be broken out): implement DRUID-based identifier schema in TACO
  - tech team walks through identifier schema
  - identifier for system different from DRUID identifiers (versioning plays into this too)?

**Comment by cmh2166** [comment]
https://github.com/sul-dlss/suri_rails

**Comment by cmh2166** [comment]
blocked by #189

**Comment by aaron-collier** [comment]
Stretch goal to write this in go to replace SURI and avoid a proxy api

**Comment by cmh2166** [comment]
https://github.com/sul-dlss-labs/id-proxy-api/blob/setup/api/identifier_api_swagger.json
